core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -142,6 +144,12 @@ angular.module('oppia').directive('adminMiscTab', [
           ctrl.dataExtractionQueryStatusMessage = message;
         };
 
+
+        ctrl.dummyMail = function() {
+          var SendMailUrl = SEND_DUMMY_MAIL_HANDLER_URL;
+          $window.open(SendMailUrl);
```

Comment:
This is not how we handle this sort of thing. See the surrounding code for details and follow the existing patterns for http requests. $window.open is only used in one specific case where the aim is to download a file.

Reply:
Done